### ISSUE TITLE
最新版本2.4.2服务下线报错errCode: 500, errMsg: do metadata operation failed

### ISSUE THREAD

**Issue by cll0228** [alibaba/nacos#12666]
最新版本的2.4.2nacos服务下线报错，log/nacos.log无报错日志
![image](https://github.com/user-attachments/assets/29d184c5-4213-4c41-87ce-5334a0fdf353)
![image](https://github.com/user-attachments/assets/01c1c12e-e9fc-4f93-b048-529749a8f78c)
![image](https://github.com/user-attachments/assets/b0b2021f-2605-46b4-a2f6-12109c87d958)

**Comment by libogithub2018** [comment]
你编辑服务不报错?  我跟了一下好像是选主的问题

**Comment by cll0228** [comment]
是的 @libogithub2018 2024-09-24 14:48:15,118 ERROR got exception. caused: errCode: 500, errMsg: do metadata operation failed ;caused: com.alibaba.nacos.consistency.exception.ConsistencyException: The Raft Group [naming_instance_metadata] did not find the Leader node;caused: The Raft Group [naming_instance_metadata] did not find the Leader node;

**Comment by cll0228** [comment]
已解决 删除 Nacos 根目录下 data 文件夹下的 protocol 文件夹即可。
